libs/pact-test-helpers/build.gradle

```diff
@@ -12,7 +12,7 @@ repositories {
 dependencies {
 	implementation "com.nimbusds:oauth2-oidc-sdk:$rootProject.ext.dependencyVersions.nimbusdsOauth2OidcSdk",
 			"au.com.dius.pact.consumer:junit5:4.6.5",
-			'com.fasterxml.jackson.core:jackson-databind:2.15.2'
+			"com.fasterxml.jackson.core:jackson-databind:2.15.3"
```

Comment:
We should _really_ add this is an explicit dependency everywhere, rather than relying on the transitive version from powertools.

This is the version currently used by powertools, and upgrading to 2.16 breaks things.